src/fsharp/CheckExpressions.fs

```diff
@@ -5358,7 +5359,18 @@ and TcExprThen cenv overallTy env tpenv synExpr delayed =
         | _ -> TcLongIdentThen cenv overallTy env tpenv longId delayed
 
     // f x
-    | SynExpr.App (hpa, _, func, arg, mFuncAndArg) ->
+    | SynExpr.App (hpa, isInfix, func, arg, mFuncAndArg) ->
+        let context =
+            cenv.tcSink.CurrentSink
```

Comment:
Using the sink is interesting as you are wanting to get access to the source text.